s/sizeof/build_info.json

```diff
@@ -0,0 +1,15 @@
+{
+    "maintainer": "vinodk99",
+    "package_name": "sizeof",
+    "github_url": "https://github.com/ehcache/sizeof",
+    "version": "master",
+    "default_branch": "master",
```

Comment:
@vinodk99 can we try build for tag https://github.com/ehcache/sizeof/releases/tag/v0.4.3